agent/crash_analyzer.py

```diff
@@ -169,25 +176,35 @@ def execute(self, result_history: list[Result]) -> AnalysisResult:
     else:
       build_script_path = ''
 
-    evaluator_lib.Evaluator.create_ossfuzz_project_with_lldb(
+    evaluator_lib.Evaluator.create_ossfuzz_project_with_gdb(
         benchmark, generated_oss_fuzz_project, fuzz_target_path, last_result,
         build_script_path, last_result.artifact_path)
 
-    self.analyze_tool = LLDBTool(benchmark,
-                                 result=last_result,
-                                 name='lldb',
-                                 project_name=generated_oss_fuzz_project)
-    self.analyze_tool.execute('compile > /dev/null')
-    # Launch LLDB and load fuzz target binary
-    self.analyze_tool.execute(f'screen -dmS lldb_session -L '
-                              f'-Logfile /tmp/lldb_log.txt '
-                              f'lldb /out/{last_result.benchmark.target_name}')
-    self.check_tool = ProjectContainerTool(
+    self.gdb_tool = GDBTool(benchmark,
+                            result=last_result,
+                            name='gdb',
+                            project_name=generated_oss_fuzz_project)
+    #TODO(maoyi): Use a dedicated debugger image, which has the binary and
+    #source code.
```

Comment:
This will take a long time to address, and require support from OSS-Fuzz.
Feel free to replace it with `TODO(dongge)`.